phenotrex/transforms/annotation.py

```diff
@@ -65,40 +67,58 @@ def fastas_to_grs(
 
     :param fasta_files: a list of DNA and/or protein FASTA files to be converted into
                         GenotypeRecords.
+    :param confidence_threshold: The threshold of confidence above which to keep an output of
+                                 deepnog annotation.
```

Comment:
```suggestion
    :param confidence_threshold: Confidence threshold of deepnog annotations below which
                                 annotations will be discarded.
```
Simply the same as above